test/integration/runtime_int_test.go

```diff
@@ -102,7 +102,7 @@ func (suite *RuntimeIntegrationTestSuite) TestInterruptSetup() {
 		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false",
 			"ACTIVESTATE_CLI_RUNTIME_SETUP_WAIT=true"),
 	)
-	time.Sleep(10 * time.Second)
+	time.Sleep(30 * time.Second)
```

Comment:
Is this related to the changes? I wouldn't expect these changes to affect installation time.

Reply:
I did not realize that the integration tests utilize a 70MB download. It was cached locally on my machine, so I did not account for download time. It is likely this test is only interrupting a download, rather than a pre-install. I want to give it a better chance of testing the right thing.